.github/workflows/test.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           - stable
     services:
       gcs:
-        image: fsouza/fake-gcs-server
+        image: brunocalza/fake-gcs-server
```

Comment:
Fixes the etag (and other fields) problem. See: https://github.com/fsouza/fake-gcs-server/compare/main...brunocalza:fake-gcs-server:main

Reply:
might be a typo here? it says: `

```Metaneration    int64                  `json:"metageneration,string"```

https://github.com/fsouza/fake-gcs-server/commit/2a1866bb287229ae6694569fc2354d801f59c7c6#diff-f7ea230e6f4870ab6dca7a2da941af6bb152b87a32d45cc971f03fd0ff2cd1b5R120

Reply:
yeah! good catch